### PR TITLE
Fixed broken links

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

v14

## Deadline (if relevant)

Anytime

### CODE REVIEW

File: 14/umbraco-commerce/release-notes/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This section contains the release notes for Umbraco Commerce 14 including all ch
 
 #### 14.1.0 (25th September 2024)
 
-* Added [Sales Tax Provider](../../key-concepts/sales-taxt-providers.md) support.
+* Added [Sales Tax Provider](../key-concepts/sales-taxt-providers.md) support.
```
Comment (sofietoft) [inline 14/umbraco-commerce/release-notes/README.md:26]: This link doesn't work in the preview 🤔

Comment (eshanrnh) [reply 14/umbraco-commerce/release-notes/README.md:26]: When I preview it opens in Github for me 😳 Not sure what's going on here. 🤷‍♀️

**Comment by eshanrnh** [comment]
I think this should fix it. The file name in Summary.md was wrong.

**Comment by sofietoft** [comment]
Oh! Sounds more like the file name should be changed instead 😅 We can do that in another PR.

Not sure why there aren't any checks for the Commerce site now though 🤔 
Could you try making some small change to re-trigger the checks to run? 🤞

**Review by sofietoft — APPROVED** [review]
Looks like it's working now 😄 👏

**Pull request merged**